lib/plugins/jexlParser.js

```diff
@@ -125,26 +125,6 @@ function expressionApplier(context, typeInformation) {
         }
 
         newAttribute.value = applyExpression(attribute.expression, context, typeInformation);
-
```

Comment:
Is there any piece of documentation about this autocast feature that should be also removed/modified?

Reply:
There is nothing about this autocast in https://github.com/telefonicaid/iotagent-node-lib/blob/master/doc/expressionLanguage.md#jexl-based-transformations

Reply:
NTC